### PR TITLE
Updates Swedish translation (Engine)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

This PR undates the translation of the messages to Swedish.

## How to test this PR

Review the translation.

### CODE REVIEW

**Comment by matsduf** [comment]
> @matsduf #1212 has just been merged and adds new messages. This PR needs to be rebased and include those new messages.

I am working on it.

File: share/sv.po [hunk 1]
```diff
@@ -272,6 +362,10 @@ msgstr "Nätåtkomst över TCP"
 msgid "AS Diversity"
 msgstr "AS-redundans"
 
+#. CONNECTIVITY:CONNECTIVITY04
+msgid "IP Prefix Diversity"
+msgstr "Redundans i IP-prefix"
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/sv.po:367]: Is it? Compare with CONNECTIVITY:CONNECTIVITY03. Is "spridning" better?

File: share/sv.po [hunk 2]
```diff
@@ -2142,6 +2285,10 @@ msgid "Test for unknown version with unknown OPTION-CODE"
 msgstr ""
 "Test med odefinierad EDNS version och med odefinierad OPTION-CODE i EDNS"
 
+#. NAMESERVER:NAMESERVER15
+msgid "Checking for revealed software version"
+msgstr "Kontrollera för avslöjad version av programvara"
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/sv.po:2290]: Updates.

File: share/sv.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattias-p) [reply share/sv.po]: Yeah, I think "spridning" is a great fit.

**Review by mattias-p — APPROVED** [review]

**Pull request merged**